### ISSUE TITLE
Set log level to debug for backup info and collection meta

### ISSUE THREAD

**Issue by ghu613** [zilliztech/milvus-backup#366]
This is not a critical issue. Tested recently and found out the log size from executing the create backup command exceeds 2MB (when number of binlog files are large).

[code.1](https://github.com/zilliztech/milvus-backup/blob/main/core/backup_impl_create_backup.go#L728)
[code.2](https://github.com/zilliztech/milvus-backup/blob/main/core/backup_impl_create_backup.go#L731)

Thanks!

**Comment by wayblink** [comment]
@ghu613 Yes, 'copy' related logs account for a large proportion. We still keep it INFO level because it helps us locate the issues when users meet some failure. It is consuming to ask open-source users change log level and retry.  We will think about it. Thanks a lot.
